operatorcert/entrypoints/ocp_version_info.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+import json
+import logging
+import pathlib
+import sys
+
+from operatorcert.utils import ocp_version_info
```

Comment:
It might not be relevant when we are calling the packages main modules, but it makes it more difficult to test them
```
➜  operator-pipelines-images git:(ISV-908) ✗ py operatorcert/entrypoints/ocp_version_info.py 
Traceback (most recent call last):
  File "operatorcert/entrypoints/ocp_version_info.py", line 7, in <module>
    from operatorcert.utils import ocp_version_info
ModuleNotFoundError: No module named 'operatorcert'
➜  operator-pipelines-images git:(ISV-908) ✗ cd operatorcert/            
➜  operatorcert git:(ISV-908) ✗ py entrypoints/ocp_version_info.py 
Traceback (most recent call last):
  File "entrypoints/ocp_version_info.py", line 7, in <module>
    from operatorcert.utils import ocp_version_info
ModuleNotFoundError: No module named 'operatorcert'
➜  operatorcert git:(ISV-908) ✗ cd entrypoints 
➜  entrypoints git:(ISV-908) ✗ py ocp_version_info.py 
Traceback (most recent call last):
  File "ocp_version_info.py", line 7, in <module>
    from operatorcert.utils import ocp_version_info
ModuleNotFoundError: No module named 'operatorcert'
```